content/en/docs/contribute/localization.md

```diff
@@ -41,13 +44,54 @@ git clone https://github.com/<username>/website
 cd website
 ```
 
-### Open a pull request
+The localization you want to help out with is inside `content/<two-letter-code>`.
 
-Next, [open a pull request](/docs/contribute/new-content/open-a-pr/#open-a-pr) (PR) to add a localization to the `kubernetes/website` repository.
+### Suggest changes
 
-The PR must include all of the [minimum required content](#minimum-required-content) before it can be approved.
+Create or update your chosen localized page based on the English original. See
+[translating content](#translating-content) for more details.
 
-For an example of adding a new localization, see the PR to enable [docs in French](https://github.com/kubernetes/website/pull/12548).
+If you notice a technical inaccuracy or other problem with the upstream (English)
+documentation, you should fix the upstream documentation first and then copy the
```

Comment:
nit: Is "copy" the right word?
... you should fix the upstream documentation first and then translate (?) the updated content into your localization.

Reply:
Reworded